xml/System.Windows.Controls/ItemsControl.xml

```diff
@@ -1531,7 +1531,7 @@ ListBox that contains multiple types of objects
   
  ]]></format>
         </remarks>
-        <related type="ExternalDocumentation" href="http://go.microsoft.com/fwlink/?LinkID=160040">Sorting and Filtering Items in a View Sample</related>
+        <related type="ExternalDocumentation" href="/previous-versions/dotnet/netframework-3.0/ms771623(v=vs.85)">Sorting and Filtering Items in a View Sample</related>
```

Comment:
this link used to link to the complete sample. so we need to try to find that at https://github.com/Microsoft/WPF-Samples. Also, don't forget to update this link in the fwlink tool since this might be used in other places. I found at least one occurrence in dotnet/docs.

Reply:
Here it is: https://github.com/Microsoft/WPF-Samples/tree/master/Data%20Binding/SortFilter

Reply:
It's updated in the fwlink tool already